Configuration/Eras/python/Era_Run2_2017_cff.py

```diff
@@ -16,10 +16,11 @@
 from Configuration.Eras.Modifier_run2_muon_2016_cff import run2_muon_2016
 from Configuration.Eras.Modifier_run2_egamma_2017_cff import run2_egamma_2017
 from Configuration.Eras.Modifier_run2_egamma_2016_cff import run2_egamma_2016
+from Configuration.Eras.Modifier_run2_tau_ul_2016_cff import run2_tau_ul_2016
 from Configuration.Eras.Modifier_ctpps_2017_cff import ctpps_2017
 from Configuration.Eras.Modifier_pixel_2016_cff import pixel_2016
 
-Run2_2017 = cms.ModifierChain(Run2_2016.copyAndExclude([run2_muon_2016, run2_HLTconditions_2016,run2_egamma_2016,pixel_2016]),
+Run2_2017 = cms.ModifierChain(Run2_2016.copyAndExclude([run2_muon_2016, run2_HLTconditions_2016,run2_egamma_2016,run2_tau_ul_2016,pixel_2016]),
```

Comment:
Weren't modifications only supposed to enter 2016 and 2018 eras?

Reply:
Yes, you are correct. But, as far as I understand the `Run2_2017` era is created by copying `Run2_2016` one excluding some 2016-specific stuff. The stuff (containing the `run2_tau_ul_2016` modifier) is listed in the `copyAndExclude([...])` expression. Is it correct?

Reply:
Indeed...
You are right, sorry Michal. I overlooked that you included it under ```copyAndExclude([...])``` in fact